stl/inc/yvals_core.h

```diff
@@ -1084,7 +1085,7 @@
 #define __cpp_lib_endian                   201907L
 #define __cpp_lib_erase_if                 201811L
 #define __cpp_lib_generic_unordered_lookup 201811L
-#define __cpp_lib_int_pow2                 201806L
+#define __cpp_lib_int_pow2                 202002L

```

Comment:
Missing a corresponding change to the feature-test macro test (and possibly a skip of the corresponding libc++ test).

Reply:
libcxx seemed to be fine.